components/status-message/src/StatusMessage.tsx

```diff
@@ -0,0 +1,47 @@
+import { FC, createElement as h } from 'react';
+import { StandardProps, classBuilder } from '@not-govuk/component-helpers';
+import { AnchorList } from "@not-govuk/anchor-list";
+
+import '../assets/StatusMessage.scss';
+
+export type StatusAction = {
+  /** Location to link to */
+  href: string
+  /** Text of the link */
+  text: string
+  /** Title of the link */
+  title?: string
+};
+
+export type StatusMessageProps = StandardProps & {
+  /** Summary of status */
+  status: string,
+  /** Extra information about the status */
+  children?: any,
+  /** Links to perform status related actions */
+  actions?: StatusAction[]
```

Comment:
`Anchor[]`

You should be able to import the type alongside `AnchorList`.

Reply:
Good shout - fixed!